python/iot3/src/iot3/core/mqtt.py

```diff
@@ -7,9 +7,10 @@
 import paho.mqtt.enums
 import paho.mqtt.packettypes
 import paho.mqtt.properties
+import ssl
 import threading
 import time
-from typing import Any, Callable, Optional, TypeAlias, Unpack
+from typing import Any, Callable, Optional, TypeAlias, Union, Unpack
```

Comment:
useless, no?

Reply:
Indeed, that's a remnant of an interim change that was eventually dropped.